R/read_rda.R

```diff
@@ -21,13 +21,6 @@
 #' # Fails
 #' try(read_rda(path))
 #' @noRd
-read_rda2 <- function(file_name) {
-  tryCatch(
-    read_rdata(file_name),
-    # Unfortunately, legacy code saved .rds files with the extension .rda
-    error = function(e) readr::read_rds(file_name)
-  )
-}
 read_rda <- function(file_name) {
   tryCatch(
     # Unfortunately, legacy code saved .rds files with the extension .rda
```

Comment:
This does not have the happy path first :( -- it wasn't passing tests and I can't refactor right now. Anyway, this was flagged as Nit so shoud be good for now.

Reply:
Also, I renamed the .R file to reflect the name of the function.